### PR TITLE
add more hints

### PR DESCRIPTION
**Pull request by ssanj**

### CODE REVIEW

File: src/main/scala/fundamentals/level03/ExceptionExercises.scala [hunk 1]
```diff
@@ -110,27 +110,19 @@ object ExceptionExercises {
...
-    * scala> createValidPeople2
+    * scala> createValidPeople
     * = List(Person("Tokyo", 30), Person("Berlin", 43))
     *
     * What issues do you run into (if any)?
     */
-  def createValidPeople2: List[Person] = ???
+  def createValidPeople: List[Person] = {
+    personStringPairs.collect {
+      case (name, age) => ???
```
Comment (felipeeflores) [inline src/main/scala/fundamentals/level03/ExceptionExercises.scala]: 👍

**Comment by wjlow** [comment]
Do you need to update the tests?

**Comment by felipeeflores** [comment]
@wjlow I think he only needed to delete one test See https://github.com/wjlow/intro-to-scala/pull/82/files#diff-d048328a104ffb04898a91ef3f52d6c2L100

**Comment by wjlow** [comment]
👍

**Pull request merged**